### ISSUE TITLE
Add a tool that looks for repeated code

### ISSUE THREAD

**Issue by k4b7** [Khan/live-editor#183]
I was answering some programming questions on KA and noticed that some students have large blocks of repeated code.  It would be cool if there was a tool that could recognize this and help students DRY.

**Comment by JavascriptFTW** [comment]
This and #182 would seem to fall into the "Advanced Options Menu" idea that I've seen discussed.

**Comment by kevinb7** [comment]
Yeah, I'm really stoked about https://github.com/Khan/live-editor/issues/182 because it'll let us try some of these ideas out without having to worry too much about the UI.
